doc/api/v8.md

```diff
@@ -212,6 +212,32 @@ v8.setFlagsFromString('--trace_gc');
 setTimeout(() => { v8.setFlagsFromString('--notrace_gc'); }, 60e3);
 ```
 
+## `v8.stopCoverage()`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+The `v8.stopCoverage()` method allows the user to stop the coverage collection
+started by [`NODE_V8_COVERAGE`][], so that V8 can release the execution count
+records and optimize code. This can be used in conjunction with
+`v8.takeCoverage()` if the user wants to collect the coverage on demand.
+
```

Comment:
It would be most helpful to have an example of these apis used together included in the docs.

Reply:
Just something to consider: Now that we have the experimental `AbortController` landed, it would make sense to start coalescing around it as a standard way of canceling things that can be canceled, including this.

For instance, if we recast this API around a `CoverageController` that extends from `AbortController`, we could have an API like:

```js
v8.coverage.abort()   // Calling the AbortController
v8.coverage.write()   // Write the currently collected coverage (alternative to v8.takeCoverage())
```

Reply:
I wonder under which circumstance would one want to abort coverage generation, though?